test/route.js

```diff
@@ -183,7 +184,7 @@ test('/documentation should redirect to /documentation/index.html', t => {
   }, (err, res) => {
     t.error(err)
     t.strictEqual(res.statusCode, 302)
-    t.strictEqual(res.headers['location'], '/documentation/index.html')
+    t.strictEqual(res.headers['location'], `${path('/', 'documentation', 'index.html')}`)
```

Comment:
Did you test this on Windows?
I'm not sure it will work.

Ps: usually we wait for at least one *LGTM* before merging a pr :)

Reply:
Right I rushed a bit, apologies for that and yes I am on windows :)

Why do you think that?

Reply:
Nevermind, read @mcollina's comment :D